source/unified-test-format/tests/Makefile

```diff
@@ -1,8 +1,8 @@
 SCHEMA=../schema-1.3.json
 
-.PHONY: all invalid valid-fail valid-pass versioned-api load-balancers gridfs HAS_AJV
+.PHONY: all invalid valid-fail valid-pass versioned-api load-balancers gridfs transcations HAS_AJV
```

Comment:
I realized that this was missed when the `mongos-unpin.yml` file was first added.

Reply:
transcations -> transactions

Reply:
Done @iwysiu